### PR TITLE
First page on landing

### PR DESCRIPTION
**Pull request by yulheekim**
Preview:
image

### CODE REVIEW

File: perf/package.json [hunk 1]
```diff
@@ -3,16 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@material-ui/core": "^1.2.2",
+    "@material-ui/core": "^1.3.1",
```
Comment (jameswhang) [inline perf/package.json]: Is there a reason u changed this?

**Comment by jameswhang** [comment]
Fails to compile.

**Comment by jameswhang** [comment]
@yulheekim Please don't just submit a PR because "it builds on my machine". It wastes people's time and is a very irresponsible thing to do. Compile error was the following:
```
Failed to compile.

./node_modules/@material-ui/core/ButtonBase/TouchRipple.js
Module not found: Can't resolve '@babel/runtime/helpers/builtin/assertThisInitialized' in '/Users/jameswhang/Documents/projects/perf_landing/perf/node_modules/@material-ui/core/ButtonBase'
```

After fixing this, I got another error. You renamed directories randomly into lowercase from capitalized case. That fucked up Git because Git doesn't know the difference between uppercase and lowercase. But JavaScript does. So your import statement in Landing.js failed to resolve. So I fixed that. 

After that, a fuckton of linter errors came up:
```
Compiled with warnings.

./src/ducks/checkout.js
  Line 1:  '_' is defined but never used      no-unused-vars
  Line 2:  'axios' is defined but never used  no-unused-vars

./src/components/CheckOut/CheckOut/CheckOut.js
  Line 3:  'connect' is defined but never used  no-unused-vars

Search for the keywords to learn more about each warning.
```

That I had to fix. 

Please don't submit PRs where the reviewer has to fix the errors and warnings for you. Review means review, not "I have all these broken crap that you need to fix for me". Again, don't assume it works everywhere just because it works on your machine. I'm not going to review any code before I can see that it at least compiles.

File: perf/src/ducks/checkout.js [hunk 2]
```diff
@@ -1,7 +1,3 @@
-import _ from 'lodash';
-import axios from 'axios';
-
-
 //Action Types
 export const GO_CHECKOUT = 'perf/checkout/GO_CHECKOUT';
```
Comment (jameswhang) [inline perf/src/ducks/checkout.js]: Why is this necessary btw? AFAIK Checkout component is not using any reducer.

File: perf/src/pages/Landing/Landing.js [hunk 3]
```diff
@@ -2,25 +2,46 @@ import React, { Component } from 'react';
...
-        </StripeProvider>
+        <section id="first" className="page">
+          <HeaderComponent/>
+          <div className="oneliner">
+            <h1>Personalized Fragrance Designed by You</h1>
+          </div>
+
+          <div className="getStartedButton">
+          <Link to="quiz" style={{"textDecoration": "none"}}>
+            <Button style={{"fontSize": "32px", "textTransform": "none"}} variant="contained" size="large">
```
Comment (jameswhang) [inline perf/src/pages/Landing/Landing.js]: Do you want to take out styles to a separate file?

Comment (jameswhang) [reply perf/src/pages/Landing/Landing.js]: @Mark-Jung Oh, you already have one. Why didn't you just use that then?

File: perf/src/pages/Landing/Landing.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ import { Elements, StripeProvider } from 'react-stripe-elements';
 import { Link } from 'react-router-dom';
 
 import { CheckOut } from '../../components/CheckOut';
-import { HeaderComponent } from '../../components/Common';
+import { HeaderComponent } from '../../components/common';
```
Comment (yulheekim) [inline perf/src/pages/Landing/Landing.js]: @Mark-Jung Is there any reason you changed this? It was fixed back in the next commit James made.

Comment (Mark-Jung) [reply perf/src/pages/Landing/Landing.js]: because in the original codebase the common directory was named 'common'. It's no longer relevant since we are using 'Common' now.

Comment (yulheekim) [reply perf/src/pages/Landing/Landing.js]: okie please check for errors before you commit :)
